test/e2e/vertical_scaling_test.go

```diff
@@ -56,7 +58,13 @@ func TestScalingUpSingleNode(t *testing.T) {
 	memberName := machineNameToEtcdMemberName(ctx, t, kubeClient, machineClient, machineName)
 	ensureHealthyMember(t, etcdClientFactory, memberName)
 
-	// TODO: step 3: clean-up: until we can scale down
+	// step 3: clean-up: delete the machine and wait until etcd member is removed from the etcd cluster
+	err = machineClient.Delete(ctx, machineName, metav1.DeleteOptions{})
+	require.NoError(t, err)
+	t.Logf("successfully deleted the machine %q from the API", machineName)
+	ensureMembersCount(t, etcdClientFactory, 3)
+	ensureMemberRemoved(t, etcdClientFactory, memberName)
+	ensureRunningMachinesAndCount(ctx, t, machineClient)
```

Comment:
> Shouldn't we first check that the machines (and nodes) have been scaled down correctly, before we can reasonably expect the member removal controller to behave correctly.

I don't think so, primary because it is the etcd operator that prevents the machine from being deleted. It needs to ensure it is safe to remove a member, then actually delete the member, and only then instruct the other controller it is okay to remove the machine.

Reply:
> because it is the etcd operator that prevents the machine from being deleted

Not today right? At least not until we have the deletion hooks controller protecting the member from machine removal.

And the preliminary member removal controller that we just added will only remove the member after the underlying machine and node are gone.
https://github.com/openshift/cluster-etcd-operator/blob/master/pkg/operator/clustermemberremovalcontroller/clustermemberremovalcontroller.go#L122-L127

But I think we could keep this order since it will become applicable once we have the expected machine hooks controller blocking machine deletion.